admin-data-explorer-slinky/src/main/scala/net/wiringbits/webapp/utils/ui/web/components/widgets/ExperimentalTableWidget.scala

```diff
@@ -19,10 +18,10 @@ import scala.util.Try
     val defaultPageLength = 10
     val defaultOffset = 0
 
-    val urlSearchParams = new URLSearchParams(dom.window.location.search)
+    val urlSearchParams = new URLSearchParams()
```

Comment:
Make sure this still works

Reply:
Now it does, didn't realize that I only had to make the change from dom.experimental to dom package